src/compute/taffy_tree.rs

```diff
@@ -279,19 +284,22 @@ fn perform_taffy_tree_hidden_layout(tree: &mut Taffy, node: NodeId) {
 ///     rather than rounding the width/height directly
 ///
 /// See <https://github.com/facebook/yoga/commit/aa5b296ac78f7a22e1aeaf4891243c6bb76488e2> for more context
-fn round_layout(tree: &mut impl LayoutTree, node: NodeId, abs_x: f32, abs_y: f32) {
-    let layout = tree.layout_mut(node);
-    let abs_x = abs_x + layout.location.x;
-    let abs_y = abs_y + layout.location.y;
+fn round_layout(tree: &mut Taffy, node_id: NodeId, abs_x: f32, abs_y: f32) {
+    let node = &mut tree.nodes[node_id.into()];
+    let unrounded_layout = node.unrounded_layout;
+    let layout = &mut node.final_layout;
+
+    let abs_x = abs_x + unrounded_layout.location.x;
+    let abs_y = abs_y + unrounded_layout.location.y;
```

Comment:
The names `abs_x` and `abs_y` feel quite unintuitive, I suspect most people would assume `abs` refers to an absolute value, not an absolute position. Maybe `parent_x` and `parent_y` would be better:

```suggestion
fn round_layout(tree: &mut Taffy, node_id: NodeId, parent_x: f32, parent_y: f32) {
    let node = &mut tree.nodes[node_id.into()];
    let unrounded_layout = node.unrounded_layout;
    let layout = &mut node.final_layout;

    // Calculate the absolute position of this node
    let abs_x = parent_x + unrounded_layout.location.x;
    let abs_y = parent_y + unrounded_layout.location.y;
```